README.md

```diff
@@ -1,31 +1,26 @@
 # flannel
 
-flannel (originally [rudder](http://comments.gmane.org/gmane.linux.coreos.devel/1683)) is an overlay network that gives a subnet to each machine for use with
-Kubernetes.
+flannel (originally [rudder](http://comments.gmane.org/gmane.linux.coreos.devel/1683)) is a virtual network that gives a subnet to each host for use with container runtimes.
```

Comment:
do we even need the rudder reference any more? flannel has gained its own reputation :-)